pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -464,6 +466,21 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	// needed for luksOpen synchronization when devices are encrypted and the osd is prepared with LVM
 	hostIPC := osdProps.storeConfig.EncryptedDevice || osdProps.encrypted
 
+	osdLabels := c.getOSDLabels(osd, failureDomainValue, osdProps.portable)
+
+	if c.spec.Network.MultiClusterService.Enabled {
+		externalIP, err := c.createOSDService(osd, osdLabels)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to configure osd service for osd.%d. %v", osd.ID)
+		} else {
```

Comment:
nit: no need for the else since there was a return statement previously. 
```suggestion
		} 
```

Reply:
I'd still suggest no need for this `else`